backend/python/app/services/implementations/story_service.py

```diff
@@ -225,25 +233,32 @@ def update_story_translation_contents(self, story_translation_contents):
 
     def get_story_translations_available_for_review(self, language, level):
         try:
-            return (
+            story_translations = (
                 db.session.query(
                     Story.id.label("story_id"),
                     Story.title.label("title"),
                     Story.description.label("description"),
                     Story.youtube_link.label("youtube_link"),
                     Story.level.label("level"),
-                    StoryTranslation.id.label("id"),
-                    StoryTranslation.language.label("language"),
-                    StoryTranslation.stage.label("stage"),
-                    StoryTranslation.translator_id.label("translator_id"),
-                    StoryTranslation.reviewer_id.label("reviewer_id"),
                 )
                 .join(StoryTranslation, Story.id == StoryTranslation.story_id)
                 .filter(Story.level <= level)
                 .filter(StoryTranslation.language == language)
                 .filter(StoryTranslation.reviewer_id == None)
                 .all()
             )
+            translationsList = []
+            for story_translation in story_translations:
+                story_translation_dict = story_translation._asdict()
+                translationsList.append(
+                    {
+                        **StoryTranslation.query.get(
```

Comment:
Is it possible to use only one query for this entire function? (and `get_story_translations` and `get_story_translation`. `get_story_translation` works but should really only be 1 query.) This might be inspiring/helpful: https://docs.sqlalchemy.org/en/14/tutorial/orm_related_objects.html#using-relationships-in-queries

You might be able to do some fancy stuff with `Story.join(StoryTranslation).join(StoryTranslationContents)`

Reply:
^ in line with this, can we look into whether there's a `.all` that exists? Seems more efficient than iterating